sys/include/net/lora.h

```diff
@@ -29,61 +29,75 @@ extern "C" {
 #endif
 
 /**
- * @name    LoRa modulation default values
+ * @defgroup net_lora_conf    LoRa modulation default values
```

Comment:
Unifying with the other configuration groups

```suggestion
 * @defgroup net_lora_conf     LoRa modulation compile configurations
```

?

Reply:
roger